README.md

```diff
@@ -1,6 +1,8 @@
 [![Travis](https://img.shields.io/travis/triplea-game/triplea.svg?style=flat-square)](https://travis-ci.org/triplea-game/triplea)[![TripleA license](https://img.shields.io/github/license/triplea-game/triplea.svg?style=flat-square)](https://github.com/triplea-game/triplea/blob/master/LICENSE)<br>
 TripleA is a free game engine that runs on open source and is community supported. 
 
+[![Code Coverage](https://img.shields.io/codecov/c/github/pvorb/property-providers/develop.svg)](https://codecov.io/github/pvorb/property-providers?branch=develop)
```

Comment:
This looks like it's pulling in the coverage badge for the https://github.com/pvorb/property-providers project instead of TripleA.

Also might want to change the badge style to `flat-square`, as I believe that's what @RoiEXLab's been using for the other badges.